docs/release-notes/2.17.0.md

```diff
@@ -0,0 +1,75 @@
+# Percona Monitoring and Management 2.17.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">May 10, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+- Custom Certificates support for MySQL monitoring, including remote monitoring, will let you monitor MySQL instances with such security restrictions. This functionality might help monitor Google Cloud SQL for MySQL instances easier.
+
+- External monitoring cleanups on UI to make it easier to use. You can fill all parameters or provide the URL of your endpoint, and PMM will detect all needed parts.
+
+- pg_stat_monitor 0.9.0 support. This change will give you compatibility with the latest version. More features coming later.
+
+- Single-line install of PMM Server on supported Linux distros (this feature is in Technical Preview).
+
+- DBaaS Changes: (this feature is in Technical Preview)
+    - DBaaS is now is better open for the public, so you can quickly turn it ON  on the Advanced settings page. ([Read more](../how-to/configure.md#advanced-settings))
+    - Database components management will enable PMM administrators to restrict some DB versions from creating DB Clusters.
+    - For PXC clusters created using DBaaS, the HAProxy will be used by default.  Please note: the proper monitoring HAProxy with DBaaS will come later.
+
+- Changes to *Sign in to Percona Platform*. From this release, Registration of the Percona account will be more secure and require additional confirmation.
+
+(???) **  Collecting data about Security Threat tool and  Integrated Alerting usage. To better understand features adoption and force our resources to needed features, we are adding and additional Telemetry data to be collected. The data will be boolean values is the feature enabled not on the PMM Server. Please note: all data is anonymous and no is PII collected.
+
+
+## New Features
+
+* [PMM-7863](https://jira.percona.com/browse/PMM-7863): [BE] Add version management for HAProxy
```

Comment:
@denisok is this makes sense now without FE part? AFAIK user can't use this for now

Reply:
7863 & 7013 have different explanations: default vs version management. If same, which is it?